src/views/common/Animation/AnimationControls/PlaybackChangeDirectionButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { faStepBackward, faStepForward } from '@fortawesome/free-solid-svg-icons'
```

Comment:
Breaking these buttons into their own components makes them easier to manage and makes the other controls way more lightweight.